sample-projects/helloworld-sample/src/ui-test/bottomBar-test.ts

```diff
@@ -41,34 +41,54 @@ describe('Bottom Bar Example Tests', () => {
             await view.getDriver().wait(() => { return problemsExist(view); }, 15000);
         });
 
+        // These tests use getAllVisibleMarkers() and are unreliable and should not be included.
+        //
         // now we can look at the error markers
-        it('Error markers are displayed', async () => {
-            // generally, there are 3 marker types (warning, error, and file - file just contains other markers though)
-            // we want to see the errors 
-            const errors = await view.getAllMarkers(MarkerType.Error);
-
-            // assert that there are errors (there should be about 8 in the file)
-            expect(errors.length).is.greaterThan(5);
-        });
-
+        // it('Error markers are displayed', async () => {
```

Comment:
I commented out these tests, as they aren't reliable.

Reply:
I think we should use it.skip(...) instead of commenting out tests. Mocha will show tests in the output but tests will not be executed. Reason comments are good and I believe we should keep them.

Reply:
Ok. I agree in using it.skip (which is good idea Marian, btw.), at the same time, we should come up with an issue, that will tackle skipped tests in such test environment (hard coded resolution, exact number of markers that are actually visible) allowing us to cover all the provided implementation.